components/user_settings/display/index.ts

```diff
@@ -36,6 +37,9 @@ function mapStateToProps(state: GlobalState) {
     const lockTeammateNameDisplay = getLicense(state).LockTeammateNameDisplay === 'true' && config.LockTeammateNameDisplay === 'true';
     const configTeammateNameDisplay = config.TeammateNameDisplay as string;
 
+    // calling it once here avoids multiple calls to the function in the return
+    const isMobile = UserAgent.isMobile();
```

Comment:
So this only applies to mobile browsers, but it won't apply to mobile web view on the desktop (which is checked by `Utils.isMobile`. Is that intended?

Reply:
No, it is not intended. Interestingly enough it works on desktop browsers with a mobile like window size as well. That's the feature I was aiming at providing

Reply:
FYI: @hmhealey I created a new UserAgent function `isMobileWeb` that checks for **NOT** being IOS or Android (`!isMobile()`) and a max screen width of 768px (defined in the constants file)

Reply:
I did change the function to check for a small viewport size only and renamed it to `isMobileViewportSize`